dataframe/testdata/dataframe_head.star

```diff
@@ -2,7 +2,7 @@ load("dataframe.star", "dataframe")
 
 
 def f():
-  df = dataframe.read_csv("""id,animal,sound
+  df = dataframe.parse_csv("""id,animal,sound
```

Comment:
this is going to break a few tests in qri. 🤷

Reply:
It will, but the fix is easy (just changing the function name). I can handle it tomorrow at the same time I dump qri's dep.